addons/medical_status/stringtable.xml

```diff
@@ -1,89 +1,87 @@
+            <German>Multiplikator um die Verblutungsgeschwindigkeit zu verändern</German>
+            <Czech>Koeficient rychlosti krvácení</Czech>
+            <Portuguese>Coeficiente para modificar a velocidade do sangramento</Portuguese>
+            <French>Modifie le débit des saignements</French>
+            <Hungarian>Egy szorzó a vérzés sebességének szabályozására</Hungarian>
+            <Italian>Coefficiente che modifica la velocità di sanguinamento</Italian>
+            <Japanese>この係数では出血速度を変更できます</Japanese>
+            <Korean>출혈의 속도를 계수만큼 변경합니다</Korean>
+            <Chinesesimp>修改流血速度</Chinesesimp>
+            <Chinese>修改流血速度</Chinese>
+        </Key>
+        <Key ID="STR_ACE_Medical_Status_PainCoefficient_DisplayName">
+            <English>Pain Coefficient</English>
+            <Russian>Коэффициент боли</Russian>
+            <Polish>Mnożnik bólu</Polish>
+            <Spanish>Coeficiente de dolor</Spanish>
+            <German>Schmerzmultiplikator</German>
+            <Czech>Koeficient bolesti</Czech>
+            <Portuguese>Coeficiente de dor</Portuguese>
+            <French>Coéfficient de douleur</French>
+            <Hungarian>Fájdalmi koefficiens</Hungarian>
+            <Italian>Coefficiente dolore</Italian>
+            <Japanese>痛みの係数</Japanese>
+            <Korean>고통 계수</Korean>
+            <Chinesesimp>疼痛系数</Chinesesimp>
+            <Chinese>疼痛係數</Chinese>
+        </Key>
+        <Key ID="STR_ACE_Medical_Status_PainCoefficient_Description">
+            <English>Coefficient for controlling the intensity of pain adjustments.</English>
+            <Russian>Коэффициент, изменяющий уровень боли</Russian>
+            <Polish>Mnożnik modyfikujący intensywność bólu</Polish>
+            <Spanish>Coeficiente para modificar la intensidad del dolor</Spanish>
+            <German>Multiplikator um die Schmerzintensität zu verändern</German>
+            <Czech>Koeficient intenzity bolesti</Czech>
+            <Portuguese>Coeficiente para modificar a instensidade de dor</Portuguese>
+            <French>Modifie l'intensité de la douleur</French>
+            <Hungarian>Egy szorzó a fájdalom erősségének szabályozására</Hungarian>
+            <Italian>Coefficiente che modifica l'intensità del dolore</Italian>
+            <Japanese>この係数では痛みの強さを変更できます</Japanese>
+            <Korean>고통의 정도를 계수만큼 변경합니다</Korean>
+            <Chinesesimp>修改疼痛强度的系数</Chinesesimp>
+            <Chinese>修改疼痛強度的係數</Chinese>
+        </Key>
+        <Key ID="STR_ACE_Medical_Status_IvFlowRate_DisplayName">
+            <English>IV Transfusion Flow Rate</English>
+            <German>Transfusions Fließrate</German>
+            <Japanese>IV 輸血の流量</Japanese>
+            <Russian>Скорость внутривенного переливания</Russian>
+        </Key>
+        <Key ID="STR_ACE_Medical_Status_IvFlowRate_Description">
+            <English>Controls how quickly IV Bags will have an effect.</English>
```

Comment:
Could use a minor rewording here (as it could be interpreted as how much time passes before the IV starts).

Reply:
Agreed. Maybe: `Controls how quickly fluid flows out of IV Bags.`? Could also describe how exactly the coefficient affects the calculation.